charts/kubeclarity/templates/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.ingress.enabled }}
```

Comment:
Ingress support is already being handled in https://github.com/openclarity/kubeclarity/pull/276, better to wait for it to be pushed.

Reply:
cool, I'll remove the ingress resource